godot-codegen/src/generator/default_parameters.rs

```diff
@@ -97,6 +97,11 @@ pub fn make_function_definition_with_defaults(
     };
 
     let functions = quote! {
+    #[doc = "To set the default parameters, use [`"]
+    #[doc = stringify!(Self::#extended_fn_name)]
+    #[doc = "`] and its builder methods.  See"]
+    #[doc = " [the book](https://godot-rust.github.io/book/godot-api/functions.html?highlight=default#default-parameters) for"]
```

Comment:
The `?highlight=default` is distracting and shouldn't be part of the link.

Reply:
fixed